bld/src/libproxy/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ set(_PROXY_COMPONENTS
     prx_err
     prx_host
     prx_module
+    prx_config
```

Comment:
This file is missing.
It looks like it was added in your repository in commit 6c1bde36faed203fcd413d2d104c70819f90b296 which is missing in this branch.